relay-server/src/endpoints/common.rs

```diff
@@ -526,6 +526,7 @@ where
                         outcome,
                         event_id: *event_id.borrow(),
                         remote_addr,
+                        event_category: None,
```

Comment:
Just checking: my assumption here is that we failed to get any items from the envelope, therefore we can't possibly supply a category. Right?